src/plugins/mer/mervirtualboxmanager.cpp

```diff
@@ -581,7 +700,48 @@ QStringList listedVirtualMachines(const QString &output)
     return vms;
 }
 
-VirtualMachineInfo virtualMachineInfoFromOutput(const QString &output)
+void fetchVdiInfo(VirtualMachineInfo &virtualMachineInfo)
+{
+    QStringList arguments;
+    arguments.append(QLatin1String(SHOWMEDIUMINFO));
+    arguments.append(virtualMachineInfo.vdiPath);
+    VBoxManageProcess process;
+    if (!process.runSynchronously(arguments))
+        return;
+
+    vdiInfoFromOutput(QString::fromLocal8Bit(process.readAllStandardOutput()), virtualMachineInfo);
+}
+
+void vdiInfoFromOutput(const QString &output, VirtualMachineInfo &virtualMachineInfo)
+{
+    //2 vdi size
+    //3 vdi size on disk
```

Comment:
It is common to put space after `//` unless it is a code snippet that is commented out